stablehlo/dialect/AssemblyFormat.h

```diff
@@ -198,6 +209,24 @@ void printCustomCallTarget(AsmPrinter& p, Operation*, StringAttr target);
 
 ParseResult parseCustomCallTarget(AsmParser& parser, StringAttr& target);
 
+// HloDim - Print an ArrayRef of integrer values with brackets.
+//
+//   Generic:
+//     1, 2
+//   Custom:
+//     [1, 2]
+FailureOr<SmallVector<int64_t>> parseHloDim(AsmParser& parser);
+void printHloDim(AsmPrinter& printer, ArrayRef<int64_t> ints);
+
+// IntArray - Calls parseHloDim, with an interface that works for
+// custom directives.
+ParseResult parseIntArray(AsmParser& parser, SmallVector<int64_t>& ints);
+
+ParseResult parseIntArray(AsmParser& parser,
+                          FailureOr<SmallVector<int64_t>>& ints);
+
+void printIntArray(AsmPrinter& printer, ArrayRef<int64_t> ints);
```

Comment:
The functions above are ordered print/parse/print/parse/.... Here the order seems to have switched to parse/print/parse/print. Let's make things consistent?